packages/agent/.mocharc.json

```diff
@@ -1,5 +1,6 @@
 {
   "enable-source-maps": true,
   "exit": true,
-  "spec": ["tests/compiled/**/*.spec.js"]
-}
\ No newline at end of file
+  "spec": ["tests/compiled/**/*.spec.js"],
+  "timeout": 10000
```

Comment:
We shouldn't be doing this in principle, will read on, but in general we should do this at per test level if we have absolutely have no choice.

Reply:
ok removed